google/async_resumable_media/_upload.py

```diff
@@ -364,10 +365,10 @@ class ResumableUpload(UploadBase):
 
     def __init__(self, upload_url, chunk_size, checksum=None, headers=None):
         super(ResumableUpload, self).__init__(upload_url, headers=headers)
-        if chunk_size % resumable_media.UPLOAD_CHUNK_SIZE != 0:
+        if chunk_size % async_resumable_media.UPLOAD_CHUNK_SIZE != 0:
```

Comment:
Should constants such as ChunkSize be duplicated, or can you use use the ones from sync?